app/components/chart-footnotes.tsx

```diff
@@ -33,6 +37,17 @@ export const ChartFootnotes = ({
     variables: { iri: dataSetIri, locale },
   });
 
+  // Data for data download
+  const filters = useQueryFilters({ chartConfig });
+  const [{ data: visibleData }] = useDataCubeObservationsQuery({
+    variables: { locale, iri: dataSetIri, dimensions: null, filters },
+  });
+  const [{ data: allData }] = useDataCubeObservationsQuery({
+    variables: { locale, iri: dataSetIri, dimensions: null, filters: null },
+  });
```

Comment:
I think it would be good to [pause](https://formidable.com/open-source/urql/docs/basics/react-preact/#pausing-usequery) those requests unless the menu has been open once. This way we do not download the data if there is no filter.

Reply:
👌